core/store/src/trie/mod.rs

```diff
@@ -25,6 +24,7 @@ use crate::trie::trie_storage::{
     TrieStorage,
 };
 use crate::{ColState, StorageError, Store, StoreUpdate};
+use near_primitives::utils::TrieKey;
```

Comment:
If you get another commit, please, fix this use statement to be in the right group.